aws/rust-runtime/aws-inlineable/src/dsql_auth_token.rs

```diff
@@ -0,0 +1,304 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+//! Code related to creating signed URLs for logging in to Xanadu.
```

Comment:
Just to check, are we allowed to mention `Xanadu` in a public GitHub repo?

Reply:
I'll update this.